### PR TITLE
[17.0][MIG] contract_sale_invoicing: Migration to version 17.0

### PR DESCRIPTION
**Pull request by juanjosesegui-tecnativa**
@Tecnativa TT50111

### CODE REVIEW

**Comment by juanjosesegui-tecnativa** [comment]
ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
Please squash administrative commits.

imagen

**Comment by juanjosesegui-tecnativa** [comment]
> Please squash administrative commits.
> 
> imagen

Done!

File: contract_sale_invoicing/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+To use this module, you need to:
+
+1.  Go to Invoicing \> Sales \> Contracts and select or create a new
```
Comment (carolinafernandez-tecnativa) [inline contract_sale_invoicing/readme/USAGE.md]: Go to Invoicing  \> Customers > Customer Contracts

File: contract_sale_invoicing/tests/test_contract_sale_invoicing.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# Copyright 2023 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from freezegun import freeze_time
+
+from odoo.addons.contract.tests.test_contract import TestContractBase
+
+
+@freeze_time("2016-02-28")
+class TestContractSaleInvoicing(TestContractBase):
```
Comment (carolinafernandez-tecnativa) [inline contract_sale_invoicing/tests/test_contract_sale_invoicing.py]: Please use BaseCommon

**Review by carolinafernandez-tecnativa — COMMENTED** [review]
Functional review LGTM :)

Still having problems when sending email in Contracts but i thinks is related with module contract base

image

Please review my comments before.

**Review by carolinafernandez-tecnativa — APPROVED** [review]

File: contract_sale_invoicing/tests/test_contract_sale_invoicing.py (hunk 2 above)
Comment (juanjosesegui-tecnativa) [reply contract_sale_invoicing/tests/test_contract_sale_invoicing.py]: Done!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**